### PR TITLE
March 2021 Update qdk-relnotes.md

### PR DESCRIPTION
**Pull request by KittyYeungQ**

### CODE REVIEW

**Review by cgranade — CHANGES_REQUESTED** [review]

File: articles/qdk-relnotes.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+## Version 0.15.2103.133969
+
+
+*Release date: March 30th, 2021*
+
+- Released QIR emission as experimental feature.
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:27]:
```suggestion
- Released QIR emission as experimental feature.
- Loosen restriction on AllowAtMostNCallsCA operation.
- Added missing APIs for Math Library.
- Removed GetQubitsAvailableToBorrow() and GetQubitsAvailableToUse()
- Fixed Q# Language Server fails during initialization in Visual Studio due to JsonReaderException.
```

File: articles/qdk-relnotes.md [hunk 2]
```diff
@@ -19,6 +19,13 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+## Version x.xx.xxxx.xxxxxx
+
+*Release date: March 30th, 2021*
+
+- Released QIR emission as experimental feature.
```
[the remark this answers is not in the document]
Comment (KittyYeungQ) [reply articles/qdk-relnotes.md:27]: @cgranade Thank you. There didn't seem to be tracked on ADO or the release planning meeting. Added below. Do we want to link to the PRs?

File: articles/qdk-relnotes.md [hunk 3]
```diff
@@ -19,6 +19,14 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+## Version 0.15.2103.133969
+
+
+*Release date: March 30th, 2021*
+
+- Released QIR emission as experimental feature.
+- Added support for multiple entry points.
```
Comment (ScottCarda-MS) [inline articles/qdk-relnotes.md:28]: @bettinaheim Should we specify that this is _command line_ support for multiple entry points, and that this isn't supported for IDE builds?

File: articles/qdk-relnotes.md (hunk 1 above)
Comment (cgranade) [reply articles/qdk-relnotes.md:27]:
```suggestion
- Released QIR emission as experimental feature.
- Loosen restriction on AllowAtMostNCallsCA operation (https://github.com/microsoft/QuantumLibraries/pull/431).
- Added missing APIs for Math Library (https://github.com/microsoft/QuantumLibraries/issues/413).
- Removed <xref:Microsoft.Quantum.Environment.GetQubitsAvailableToBorrow> and <xref:Microsoft.Quantum.Environment.GetQubitsAvailableToUse> (https://github.com/microsoft/QuantumLibraries/issues/418).
- Fixed Q# Language Server fails during initialization in Visual Studio due to JsonReaderException (https://github.com/microsoft/qsharp-compiler/issues/885).
```

File: articles/qdk-relnotes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cgranade) [reply articles/qdk-relnotes.md:27]: Thanks for bringing these in from the links in our March 2021 milestones. Please see suggested edits below.

**Review by cgranade — APPROVED** [review]

**Pull request merged**